.github/workflows/validate-pr-release-info.yml

```diff
@@ -19,10 +19,20 @@ env:
   IS_LABELED_NON_LOGICAL: contains(github.event.pull_request.labels.*.name, 'tooling') || contains(github.event.pull_request.labels.*.name, 'documentation') || contains(github.event.pull_request.labels.*.name, 'restructure')
 
 jobs:
+  get-env-vars:
+    name: Load Environment Vars Into Outputs
+    runs-on: ubuntu-latest
+    outputs: 
+      IS_LABELED_RELEASE: ${{ env.IS_LABELED_RELEASE }}
+      IS_LABELED_NON_LOGICAL: ${{ env.IS_LABELED_NON_LOGICAL }}
+    steps:
+      - run: echo "null"
```

Comment:
Why `echo "null"` here?

Reply:
Ah in that case can there be a comment or something in the echo message which indicates that?